plutus-ledger/src/Ledger/Scripts.hs

```diff
@@ -84,10 +86,22 @@ import           LedgerBytes                         (LedgerBytes (..))
 -- | A script on the chain. This is an opaque type as far as the chain is concerned.
 newtype Script = Script { unScript :: UPLC.Program UPLC.DeBruijn PLC.DefaultUni PLC.DefaultFun () }
   deriving stock Generic
-  deriving Serialise via UPLC.OmitUnitAnnotations UPLC.DeBruijn PLC.DefaultUni PLC.DefaultFun
+  deriving newtype Flat
```

Comment:
I think we probably *don't* want this: you shouldn't be using `Flat` to work with `Script`, you should be using `CBOR`.

Reply:
And it's gone.